autoload/vp4.vim

```diff
@@ -137,7 +161,7 @@ function! s:PerforceFstat(field, filename)
 
     " Extract the value from the string which looks like:
     "   ... headRev 65\n\n
-    let val = split(split(s, '\n')[0], ' ')[2]
+    let val = split(split(substitute(s, '\r', '', ''), '\n')[0])[2]
```

Comment:
not sure if there's an OS agnostic way to do this.. but this works!

Reply:
I did a quick search and couldn't find anything :(